cmd/kyverno/main.go

```diff
@@ -92,13 +92,19 @@ var (
 	admissionReports           bool
 	reportsChunkSize           int
 	logFormat                  string
+	dumpPayload                bool
+	dumpOperations             string
```

Comment:
We should keep only `dumpPayload` for now

Reply:
alright